#!/usr/bin/python
PACKAGE = 'openface_classifier'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("classifier_path",    str_t,    0, "Classifier Path",  "/home/jaskaran/train_images/classifier.pkl")
gen.add("feature_path", str_t, 0, "Features Path", '/home/jaskaran/train_images/features.npy')

exit(gen.generate(PACKAGE,'openface_classifier','classifierConfig'))